docs/operations/export_api.md

```diff
@@ -217,6 +217,23 @@ The export server certificate is valid for 7 days after which it is rotated by d
 #### External link certificates
 The external link certificates are associated with the Ingress/Route that points to the service created by the KubeVirt operator. The CA that signed the Ingress/Route will part of the certificates. 
 
+### TTL for an Export
```

Comment:
I would fully spell out TimeToLive (TTL) in the header, so people unfamiliar with the acronym can still understand what it means.

Reply:
done